docs/reference/pages/model_reference.rst

```diff
@@ -356,6 +386,89 @@ Methods and properties
 
     .. automethod:: get_site_root_paths
 
+
+Locale
+======
+
+The Locale model defines the set of locales that can be used on a site.
+Each ``Locale`` record corresponds to a language code defined in the ``WAGTAIL_CONTENT_LANGUAGES`` setting.
+
+Wagtail will initially set up one ``Locale`` to act as the default language for all existing content.
+This first locale will automatically pick the value from ``WAGTAIL_CONTENT_LANGUAGES`` that most closely matches the sites primary language code defined in ``LANGUAGE_CODE``.
+If the primary language code is changed later, Wagtail will not automatically create a new ``Locale`` record or update an existing one.
```

Comment:
```suggestion
If the primary language code is changed later, Wagtail will **not** automatically create a new ``Locale`` record or update an existing one.
```

imho, would be good to emphasise the "not" part otherwise it can could be missed